### PR TITLE
Move recogniser registry to recogniser directory

### PR DESCRIPTION
**Pull request by gabechu**
### Description
Relocate recogniser registry and create initialisation under `pii_recognition.recognisers.__init__.py`.

### Context
Items like recognisers and tokenisers were used to register at the root init file `__init__.py` all at once. In this implementation we cannot create separate registries for different collections of items. That is, we cannot have a `tokeniser_regitry` without instantiating a `recogniser_registry`. Besides, registry would know too much about every item it needs to register, meaning you have to import all of the parent class of the registered items to the registry folder.

### Solution
The solution is to have recognisers registered under `recognisers.__init__.py`. Applying the same logic, tokenisers will be registered under `tokenisation.__init__.py` etc.

### Todos
- [ ] Do we want to test `pii_recognition/recognisers/__init__.py`? How to efficiently test it since direct testing checking if a certain recogniser gets registered could be slow, `import flair` would take time.

### CODE REVIEW

**Comment by gabechu** [comment]
@ePak For the function `init()` within the `__init__`file, do we want to test it? I feel unittest may not be appropriate. Does it mean we are going to leave to integration test? Or it's okay to not test it?

File: pii_recognition/evaluation/pipeline_test.py [hunk 1]
```diff
@@ -3,17 +3,21 @@
...
-        pass
+class Simple:
+    pass
 
-    class Complex:
-        def __init__(self, a):
-            self.a = a
 
-    mock_regsitry.recogniser = {"Simple": Simple, "Complex": Complex}
+class Complex:
```
Comment (kerrychu-zen) [inline pii_recognition/evaluation/pipeline_test.py:10]: Can we give more meaningful names other than just `Simple` and `Complex` pls?

Comment (gabechu) [reply pii_recognition/evaluation/pipeline_test.py]: For sure! I've rename to `RegistryNoConfig` and `RegistryWithConfig` along with a line of comment to each renamed class. Let me know if it makes sense.

Comment (kerrychu-zen) [reply pii_recognition/evaluation/pipeline_test.py]: makes much more sense now! 👍

**Pull request merged**